draft-ietf-quic-http.md

```diff
@@ -1106,6 +1131,13 @@ PUSH_PROMISE in relation to certain parts of the response is important (see
 Section 8.2.1 of {{!RFC7540}}).  Promised requests MUST conform to the
 requirements in Section 8.2 of {{!RFC7540}}.
 
+The same server push can be associated with additional client requests using a
+DUPLICATE_PUSH frame (see {{frame-duplicate-push}}).  Ordering of a
+DUPLICATE_PUSH in relation to certain parts of the response is similarly
+important.  Due to reordering, DUPLICATE_PUSH frames can arrive before the
+corresponding PUSH_PROMISE frame, in which case the request headers of the push
+might not be immediately available.
```

Comment:
Why "might not?"  If _DUPLICATE_PUSH_ arrives before the _PUSH_PROMISE_ it references, the request headers **won't** be available.

Reply:
I was thinking conditionally, "might not be available because DP could arrive before PP," but since this is inside the hypothetical, you're correct.